### PR TITLE
Sorted package loading

### PR DESCRIPTION
**Pull request by philmikejones**
Amended package loading as you suggested in previous comments.
Added tidyr (missing but required by build.R)
Added liblapack etc packages necessary to make ipfp in Ubuntu

### CODE REVIEW

File: 02-what-is-smsim.Rmd [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "What is spatial microsimulation?"
 layout: default
-output: word_document
```
Comment (Robinlovelace) [inline 02-what-is-smsim.Rmd]: Reasonable - individual chapters look nicer as pdfs!

File: 08-validation.Rmd [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "CakeMap"
+title: "Validation"
```
Comment (Robinlovelace) [inline 08-validation.Rmd]: Ah yes - schoolboy error on my part. Note the titles wash away when the book is built

File: README.md [hunk 3]
```diff
@@ -23,7 +23,14 @@ git clone https://github.com/Robinlovelace/spatial-microsim-book.git
...
 ```
-install.packages("knitr", "rmarkdown", "png", "ggmap", "dplyr", "ipfp", "rgeos", "mipfp", "rgdal", "gridExtra") 
+pkgs <- c("knitr", "rmarkdown", "png", "ggmap", "dplyr", "ipfp", "rgeos", "mipfp", "rgdal", "gridExtra", "tidyr", "mlogit")
+install.packages(pkgs)
+```
+
+To install `ipfp` on Ubuntu (tested with version 14.04, the latest 'long-term support' version at the time of writing) a number of additional packages are required. These can be installed from the command line with:
+
+```
+sudo apt-get install liblapack-dev liblapack3 libopenblas-base libopenblas-dev
```
Comment (Robinlovelace) [inline README.md]: cool - I didn't know it needed all that! blas is fast.

Comment (philmikejones) [inline README.md]: Actually I've just double-checked the requirements and the *blas packages aren't required. I'm amend for a future pull request. The liblapack* packages are needed though...

File: 08-validation.Rmd (hunk 2 above)
Comment (philmikejones) [inline 08-validation.Rmd]: There's bound to be at least one in a project of this size! :-)

**Comment by Robinlovelace** [comment]
:+1:

File: 02-what-is-smsim.Rmd (hunk 1 above)
Comment (philmikejones) [inline 02-what-is-smsim.Rmd]: I hadn't realised these had all changed! I'd just knitted them as pdfs rather than Word because I don't have Word.

**Pull request merged**